packages/commons/src/antons-pet.class.ts

```diff
@@ -0,0 +1,7 @@
+export default class AntonsPet {
+  name: string;
+
+  constructor(name: string) {
```

Comment:
maybe remove the constructor if we still don't use it.

Reply:
resolved by cefce26b6eaabb24c00da569b8f89a7fa9dcb5ab